WebImage/SDMapKitWebImage.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * This file is part of the SDWebImage package.
+ * (c) Olivier Poitrey <rs@dailymotion.com>
+ * (c) Florent Vilmart
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+#import <MapKit/MapKit.h>
+
+//! Project version number for WebImage.
+FOUNDATION_EXPORT double MapKitWebImageVersionNumber;
+
+//! Project version string for WebImage.
+FOUNDATION_EXPORT const unsigned char MapKitWebImageVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <MapKitWebImage/PublicHeader.h>
+
+// MapKit
+#if __has_include(<SDWebImage/MKAnnotationView+WebCache.h>)
+#import <SDMapKitWebImage/MKAnnotationView+WebCache.h>
```

Comment:
Emm...This import seems contains issue. Maybe we should use `#if __has_include(< SDMapKitWebImage/MKAnnotationView+WebCache.h>)`

Any, I still suggest to use `SDWebImageMapKit` for generated framework naming. Using the suffix one seems a little strange compared to other frameworks.

Reply:
Agree with @dreampiggy . `SDWebImageMapKit` maybe better IMO.

Reply:
Ok. I will make the change. Not too happy about it :))

Reply:
You are correct about the import. It's strange, but probably because the other path exists too, it works.